tests/test_signer.py

```diff
@@ -703,6 +703,21 @@ def test_generate(self):
             with self.assertRaises(UnverifiedSignatureError):
                 signer.public_key.verify_signature(sig, b"NOT DATA")
 
+    def test_private_bytes(self):
+        """Test private_bytes -> from_priv_key_uri"""
+        for pem in ["rsa", "ecdsa", "ed25519"]:
+            with open(PEMS_DIR / f"{pem}_private.pem", "rb") as f:
+                privkey = load_pem_private_key(f.read(), None)
+                signer = CryptoSigner(privkey)
+
+            with open("privkey.pem", "wb") as f:
+                f.write(signer.private_bytes)
```

Comment:
Should we write this to a temp/test dir? Otherwise this litters cwd.

Reply:
oh true. I wish this was a default unittest thing...